rand_chacha/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-- impl PartialEq+Eq for ChaChaXRng and ChaChaXCore (#975)
+## [0.3.0] - 2020-12-07
+- Bump `rand_core` version to 0.6.0
+- Bump MSRV to 1.36 (#1011)
+- Remove usage of deprecated feature "simd" of `ppv-lite86` (#979), revert
```

Comment:
This is a bit confusing. What exactly is reverted?

Reply:
The previous change (removal of feature usage)

Reply:
Maybe write 'Use deprecated feature "simd" of `ppv-lite86` for improved compatibility'?